server/models/indexdb.js

```diff
@@ -0,0 +1,16 @@
+import pg from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
+const connectionString = 'postgres://postgres:temitope@localhost:5432/ridemyway';
```

Comment:
If I'm not mistaken, you pushed your database password online right?

Reply:
yes i did .. I will fix that